packages/ember-debug/lib/error.js

```diff
@@ -18,15 +7,12 @@ function ExtendBuiltin(klass) {
   @constructor
   @public
 */
-export default class EmberError extends ExtendBuiltin(Error) {
-  constructor(message) {
-    super();
 
-    if (!(this instanceof EmberError)) {
-      return new EmberError(message);
-    }
+export default EmberError;
+function EmberError(message, code) {
+  if (this instanceof EmberError) {
+    let error = Error.call(this, message, code);
```

Comment:
I'll open another PR that deprecated the call constructor behavior so by the best LTS we can drop